src/Adapters/Storage/Postgres/PostgresStorageAdapter.js

```diff
@@ -1499,6 +1499,14 @@ export class PostgresStorageAdapter {
       if (countField) {
         results[0][countField] = parseInt(results[0][countField], 10);
       }
+      results.forEach(result => {
+        if (result.hasOwnProperty('_id')) {
```

Comment:
Can’t we use the usual transformers here?

Reply:
I was just about to change it

Reply:
I don't think we can use the usual transformers like `buildWhereClause`. We might be able to reuse skip, limit, sort that is in `find()`. It not complete yet but I could try to refactor it in the second iteration